src/Application/index.js

```diff
@@ -119,6 +119,32 @@ export default function(App, appData, platformSettings) {
       Accessibility.colorshift(this, type, config)
     }
 
+    get keymapping() {
+      return this.stage.application.config.keys
+    }
+
+    /**
+     * This function overides the default keymap with the latest keymap.
+     * @param customKeymap
+     * @param keepDuplicates
+     */
+    overRideKeyMap(customKeymap, keepDuplicates = false) {
```

Comment:
Why is the word override capitalized like this?

Reply:
Thanks @ricardo-a-alves-alb for pointing out that, we have updated the function name as well as some variables names to make it all consistent.